Update app-service-linux-intro.md

jhenriks79 opened the pull request:
Updating article to ensure web apps naming is consistent and cleanup some of the long bullet lists.

jhenriks79 committed:
Update app-service-linux-intro.md

Updating article to ensure web apps naming is consistent and cleanup some of the long bullet lists.

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -1,6 +1,6 @@
 ---
-title: Introduction to Azure Web App on Linux | Microsoft Docs
-description: Learn about Azure Web App on Linux.
+title: Introduction to Azure Web Apps on Linux | Microsoft Docs
```
The Product name is **Web App on Linux**

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -1,6 +1,6 @@
-title: Introduction to Azure Web App on Linux | Microsoft Docs
-description: Learn about Azure Web App on Linux.
+title: Introduction to Azure Web Apps on Linux | Microsoft Docs
+description: Learn about Azure Web Apps on Linux.
```
Web App on Linux

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 
 ---
-# Introduction to Azure Web App on Linux
+# Introduction to Azure Web Apps on Linux
```
Web App on Linux

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
-	* 2.3
-
-Customers can deploy their applications by using:
+[Web Apps](https://docs.microsoft.com/en-us/azure/app-service-web/app-service-web-overview) is a fully managed compute platform that is optimized for hosting websites and web applications. Customers can use Web Apps on Linux to host web apps natively on Linux for supported application stacks. The following sections lists the application stacks that are currently supported. 
```
Web App is the proper product name.

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
+|      4.4, 4.5      | 5.6 |  1.0-1.1  |  2.3 |
+| 6.2, 6.6, 6.9-6.11 | 7.0 |           |      |
+|       8.0-8.1      |     |           |      |
+
```
You could collapse this table to remove the whitespace as it isn't significant and while it does look nice from the source view, it's actually harder to update.

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 
 ## Limitations
-The Azure portal shows only features that currently work for Web App on Linux and hides the rest. As we enable more features, they will be visible on the portal.
+The Azure portal shows only features that currently work for Web Apps on Linux. As we enable more features, they will become visible on the portal.
```
Web App on Linux

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 
-For application scaling:
+### DevOps
 
```
We aren't able to use acronyms as customers may not be familiar with all of our product offerings. Please expand ACR out to [Azure Container Registry](https://docs.microsoft.com/en-us/azure/container-registry/container-registry-intro)

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 
-For application scaling:
+### DevOps
 
```
It would be helpful to customers to be able to address how Web App on Linux provides benefits to customers in regards to DevOps.

jhenriks79 replied:
Don't we want to show what features are available rather than customer benefits?

SyntaxC4 replied:
Yes - but to just iterate them on a list doesn't provide much information.

jhenriks79 replied:
Totally agree. I wasn't sure if that section was there for a reason. Doesn't sound like that's the case. Should we remove?

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 
 Some features, such as virtual network integration, Azure Active Directory/third-party authentication, or Kudu site extensions, are not available yet. Once these features are available, we will update our documentation and blog about the changes.
 
 This public preview is currently only available in the following regions:
```
It would be helpful to provide a link to the [Azure Status Dashboard](https://azure.microsoft.com/status) to let people know which regions will be available.

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
+| West US          | Southeast Asia |              |
 
-Web Apps on Linux is only supported in the Dedicated app service plans and does not have a Free or Shared tier. Also, App Service plans for regular and Linux web apps are mutually exclusive, so you cannot create a Linux web app in a non-Linux app service plan.
+Web Apps on Linux is only supported with [Basic, Standard, and Premium](https://azure.microsoft.com/en-us/pricing/details/app-service/plans/) app service plans and does not have a [Free or Shared](https://azure.microsoft.com/en-us/pricing/details/app-service/plans/) tier. The following are also important restrictions for Web Apps on Linux:
```
Web App on Linux

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
+Web Apps on Linux is only supported with [Basic, Standard, and Premium](https://azure.microsoft.com/en-us/pricing/details/app-service/plans/) app service plans and does not have a [Free or Shared](https://azure.microsoft.com/en-us/pricing/details/app-service/plans/) tier. The following are also important restrictions for Web Apps on Linux:
 
-Web Apps on Linux must be created in a resource group that does not contain non-Linux web apps in the same region.
+* You cannot create Web Apps on Linux in an App Service plan already hosting non-Linux Web Apps.
```
Web App on Linux

SyntaxC4 commented on articles/app-service-web/app-service-linux-intro.md:
```diff
@@ -18,82 +18,63 @@ ms.date: 02/16/2017
 
-Web Apps on Linux must be created in a resource group that does not contain non-Linux web apps in the same region.
+* You cannot create Web Apps on Linux in an App Service plan already hosting non-Linux Web Apps.
+* When creating Web Apps on Linux in a resource group containing non-Linux Web Apps, you must create an App Service plan in a different region than the existing App Service plan.
```
Web App on Linux

SyntaxC4 reviewed (changes requested):
Hi @jhenriks79,

Thank you for your contribution, could you please review the comments below and address them in your PR?

Thanks,

Cory Fowler
@SyntaxC4

jhenriks79 committed:
Update app-service-linux-intro.md

Changing to Web App on Linux, removing white space from tables, and adding links.
